### PR TITLE
Add amplitude event to code.org/music and update activity links

### PR DESCRIPTION
**Pull request by kelbyhawn**
Adds Amplitude (and Statsig) events to https://code.org/music, and updates the existing urls to the activity to include a `/reset` at the end in a few places.

## Links
Jira ticket: [ACQ-1869](https://codedotorg.atlassian.net/browse/ACQ-1869)

## Testing story
Local testing

----

Screenshot 2024-05-07 at 2 19 38 PM

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/music/index.haml [hunk 1]
```diff
@@ -86,3 +86,5 @@ theme: responsive_full_width
       =hoc_s(:dance_afe_disclaimer)
 
 = view :swiper_page_music_lab
+
+= view :analytics_event_log_helper, event_name: AnalyticsConstants::MUSIC_PAGE_VISITED_EVENT
```
Comment (hannahbergam) [inline pegasus/sites.v3/code.org/public/music/index.haml:90]: Is the analytics event log helper automatically sending all events to both?

Comment (hannahbergam) [reply pegasus/sites.v3/code.org/public/music/index.haml:90]: I just double checked- that .js file would require a refactor to send Statsig events by default

Comment (hannahbergam) [reply pegasus/sites.v3/code.org/public/music/index.haml:90]: We can pair on it this PM if you want! (Also... if Statsig isn't a requirement here then please ignore me)

Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/music/index.haml:90]: I don't know how that Statsig console message originally showed up, and I can't get it to show up again, weird! 🤔  

@dmcavoy do you want Statsig set up for Pegasus pages too?

Comment (dmcavoy) [reply pegasus/sites.v3/code.org/public/music/index.haml:90]: Not right now. Lets keep this just amplitude

**Review by hannahbergam — COMMENTED** [review]
The screenshot is of two user properties events, not 'page visited' events. You should be able to see the event show up in the console under the two you showed (which happen on nearly every page load). Can you update the screenshot with the specific events?

**Review by hannahbergam — APPROVED** [review]
I see the new screenshot yay!

**Pull request merged**